mitodl/odl-video-service
# Use sentry sdk instead of raven

#### What are the relevant tickets?
Closes #868 

#### What's this PR do?
- Replaces raven with sentry-sdk
- Removes excess JS entry files & renames 'CollectionApp.js' to 'App.js'

#### How should this be manually tested?
- Set SENTRY_DSN in your .env file (you can get the value from heroku odl-video-ci)
- Hardcode an error in python and JS, hit that code, verify it shows up in sentry (need to use a URL like `ovs.odl.local` instead of `localhost`)
- Verify that all URL's still work (`/collections`, `/collections/<collection_id>`, `/videos/<video_id>`, `/videos/<video_id>/embed`, `/help`, `/terms`)
